countUp

```diff
@@ -0,0 +1,1064 @@
+site/app/controllers/AbstractController.php:            return $gradeable->getComponent($component_id);
```

Comment:
Please remove this file, it appears to be a diff that was checked in with you commits.